dubdubdub/settings.py

```diff
@@ -298,6 +298,10 @@
     'enabled_methods': ['get', ]
 }
 
+# Temporary hardcoded IDS for the OLP and GLP bits.
+STATE_STARTING_SCHOOL_ID = 63929
+STATE_STARTING_BOUNDARY_ID = 11042
```

Comment:
The way that the `populateOLP` script gets the data in is by calculating the latest `School` and `Boundary` IDs currently in the db, then adding 1 to it and then incrementing the primary_key by 1 for each new School or Boundary being created. So these values above are the "atest `School` and `Boundary` IDs currently in the db".